src/components/admin/form-components/BoundRequiredTextField.jsx

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import { capFirstLetter } from 'lib/utilities';
+
 import { RequiredTextField } from 'components/admin/form-components/validated-fields';
 
-class TitleField extends React.Component {
+export class BoundRequiredTextField extends React.Component {
   constructor(props) {
     super(props);
     this.onChange = event => props.onUpdate(event.target.value);
```

Comment:
I thought by Bound you meant that it had a max length, no? Doesn't seem to be implemented

Reply:
The bounded has nothing to do with the max length.